browser-test/src/trusted_intermediary.test.ts

```diff
@@ -668,6 +668,66 @@ test.describe('Trusted intermediaries', () => {
     await validateScreenshot(page, 'manage-ti-group-members-page')
   })
 
+  test('sort trusted intermediaries based on selection', async ({
+    page,
+    adminTiGroups,
+  }) => {
+    await loginAsAdmin(page)
+    // add "aaa" with 3 members
```

Comment:
suggestion (non-blocking): use `test.step` to avoid needing code comments and make it easier to debug failures: https://playwright.dev/docs/api/class-test#test-step